auth_brute_force/models/res_authentication_attempt.py

```diff
@@ -79,16 +98,19 @@ def _hits_limit(self, limit, remote, login=None):
             domain += [("login", "=", login)]
         # Find last successful login
         last_ok = self.search(
-            domain + [("result", "=", "successful")],
+            domain + [("result", "in", ["successful", "unbanned"])],
             order='create_date desc',
             limit=1,
         )
+
         if last_ok:
             domain += [("create_date", ">", last_ok.create_date)]
         # Count failures since last success, if any
         recent_failures = self.search_count(
-            domain + [("result", "!=", "successful")],
-        )
+            domain + [
+                ("result", "!=", "successful"),
+                ("result", "!=", "unbanned"),
```

Comment:
Use `not in`, just like above.

Reply:
`res.authentication.attempt` has null value on `result` when it is created. In order to not read this record, I added the value False in the not in. This behaviour is related to [odoo not in management](https://github.com/odoo/odoo/blob/11.0/odoo/osv/expression.py#L1209).